sopel/coretasks.py

```diff
@@ -90,11 +137,15 @@ def startup(bot, trigger):
 
     if bot.config.core.throttle_join:
         throttle_rate = int(bot.config.core.throttle_join)
+        throttle_wait = max(bot.config.core.throttle_wait, 1)
```

Comment:
Do you intend to prevent someone from using `throttle_wait = 0` if they explicitly set it to that value?

Reply:
Yup, hence the `max(setting, 1)`: there is no point having a `throttle_join` with 0s of wait time.